docs/guides/user-guides/user-quickstart-guide.md

```diff
@@ -24,17 +24,18 @@ Before using [OpenMS.org](https://www.openms.de/), you need to be familiar with
 
 ## How to run a Tool
 
-It is recommended to use TOPPAS. A good start are the example pipelines (select **File** > **Open example file** within TOPPAS).
-In parallel read the documentation of the tools (see [TOPP tutorial](), [TOPP documentation](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html)) and the one of TOPPAS ([TOPPAS tutorial](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html)).
+A good start are the example pipelines (select **File** > **Open example file** within TOPPAS).
 
-Alternatively, you can use the command line and call tools directly. In this case you'll probably want to use some type of shell
+Read the documentation of the tools (see [TOPP tutorial](), [TOPP documentation](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html)) and the one of TOPPAS ([TOPPAS tutorial](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html)).
+
+Alternatively, use the command line and call tools directly. In this case, you'll probably want to use some type of shell
 script for automation.
 
-## Adapting your Pipeline Parameters
+## Adapting your pipeline parameters
```

Comment:
Adapt your pipeline parameters. For reference, view [this](https://developers.google.com/style/headings#heading-and-title-text).